readme-gen.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+SCRIPTPATH=$(dirname "$0")
+README_GENERATOR_VERSION="2.5.1"
+README_GENERATOR_EXE="readme-generator"
+
+# platform agnostic npm install, also adds into the path
+npm install -g "@bitnami/readme-generator-for-helm@${README_GENERATOR_VERSION}"
```

Comment:
This requires people to already have `npm` installed, so we should probably check if the npm executable is available, if not we can print a message that npm has to be installed.

Reply:
Done.